### PR TITLE
#10 GRPC controller

### PR DESCRIPTION
**Pull request by alex-a-renoire**

### CODE REVIEW

File: pkg/grpc/controller/controller.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	client pb.GrpcServiceClient
+}
+
+func New(cli pb.GrpcServiceClient) GRPCСontroller {
+	return GRPCСontroller{
+		client: cli,
+	}
+}
+
+type Item struct {
```
Comment (DrIhor) [inline pkg/grpc/controller/controller.go:22]: May be better to separate as model(general or other file)

**Comment by romanilchyshyn** [comment]
Am I understand right that `pb.GrpcServiceClient` connection to server will be handled at cmd exec layer?

File: pkg/grpc/controller/controller.go (hunk 1 above)
Comment (alex-a-renoire) [reply pkg/grpc/controller/controller.go:22]: I will make a unified model when all my pull requests are merged.

**Comment by alex-a-renoire** [comment]
> Am I understand right that `pb.GrpcServiceClient` connection to server will be handled at cmd exec layer?

Yes.

**Review by romanilchyshyn — COMMENTED** [review]
LGTM

**Review by DrIhor — APPROVED** [review]

**Review by stasBigunenko — APPROVED** [review]

**Pull request merged**